retrofit-adapters/rxjava/src/main/java/retrofit/HttpException.java

```diff
@@ -1,9 +1,7 @@
 package retrofit;
 
-import java.io.IOException;
-
 /** Exception for an unexpected, non-2xx HTTP response. */
```

Comment:
I don't think it has a reason to. It always delivers non-200s to response callback or returned from execute. The need for this arises only out of a user wanting 200s sent to 'onNext' and everything else needing to go to 'onError'.